### PR TITLE
Add wealth and consumption imputations

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
Adds new variables from https://github.com/PolicyEngine/openfisca-uk-data/issues/59 to PolicyEngine.

### CODE REVIEW

File: policyengine-client/src/countries/uk/data/situation.jsx [hunk 1]
```diff
@@ -68,10 +83,25 @@ export const VARIABLE_CATEGORIES = {
         "weekly_hours",
     ],
     "Wealth": [
+        "owned_land_value",
+        "property_wealth",
+        "corporate_wealth",
         "land_value",
```
Comment (MaxGhenis) [inline policyengine-client/src/countries/uk/data/situation.jsx:89]: should this be `estimated_land_value`? and similarly below, `estimated_carbon_emissions`?

could be separate from this pr

File: policyengine/countries/uk/default_reform/reform.py [hunk 2]
```diff
@@ -87,14 +95,77 @@ def create_default_reform() -> ReformType:
...
+        entity = Household
+        label = "Wealth tax"
+        documentation = "Flat tax on net financial wealth"
+        unit = "currency-GBP"
+        definition_period = YEAR
+        value_type = float
+
+        def formula(household, period, parameters):
+            rate = parameters(period).reforms.wealth_tax.rate
+            return household("net_financial_wealth", period) * rate
```
Comment (MaxGhenis) [inline policyengine/countries/uk/default_reform/reform.py:140]: I suspect we'll want `max_(0, ...)` here

File: policyengine/countries/uk/default_reform/reform.py [hunk 3]
```diff
@@ -113,6 +280,12 @@ class carbon_consumption(Variable):
         definition_period = YEAR
         value_type = float
 
+        def formula(household, period):
```
Comment (MaxGhenis) [inline policyengine/countries/uk/default_reform/reform.py:283]: while you're here, unit should be `CO2` not `C02`

File: policyengine/countries/uk/reform_parameters.yaml [hunk 4]
```diff
@@ -23,6 +23,30 @@ reforms:
...
+  wealth_tax:
+    rate:
+      description: A yearly flat tax on net financial wealth
+      values:
+        2021-01-01: 0.00
+      metadata:
+        label: Wealth Tax
+        name: wealth_tax
+        unit: /1
+        max: 0.1
```
Comment (MaxGhenis) [inline policyengine/countries/uk/reform_parameters.yaml:35]: Just a note that this seems like a good max. Bernie Sanders has the highest proposal I'm aware of, and his top rate is 8%.

File: policyengine-client/src/countries/uk/data/situation.jsx (hunk 1 above)
Comment (nikhilwoodruff) [reply policyengine-client/src/countries/uk/data/situation.jsx:89]: Added to the labels, rather than the variable names, is that OK?

**Pull request merged**